src/AggregateFunctions/AggregateFunctionRankCorrelation.h

```diff
@@ -32,7 +32,7 @@ struct RankCorrelationData : public StatisticalSample<Float64, Float64>
         std::tie(ranks_y, std::ignore) = computeRanksAndTieCorrection(this->y);
 
         /// Sizes can be non-equal due to skipped NaNs.
-        const auto size = std::min(this->size_x, this->size_y);
+        const Float32 size = static_cast<Float32>(std::min(this->size_x, this->size_y));
```

Comment:
I think that if we were to use Float in this case - it is better to use Float64 :)
```suggestion
        const Float64 size = static_cast<Float64>(std::min(this->size_x, this->size_y));
```